pypesto/store/read_from_hdf5.py

```diff
@@ -120,3 +150,94 @@ def read(self) -> Result:
                 self.results.optimize_result.append(result)
             self.results.optimize_result.sort()
         return self.results
+
+
+class SamplingResultHDF5Reader:
+    """
+    Reader of the HDF5 result files written
+    by class SamplingResultHDF5Writer.
+
+    Attributes
+    -------------
+    storage_filename:
+        HDF5 result file name
+    """
+    def __init__(self, storage_filename: str):
+        """
+        Parameters
+        ----------
+
+        storage_filename: str
+            HDF5 result file name
+        """
+        self.storage_filename = storage_filename
+        self.results = Result()
+
+    def read(self) -> Result:
+        """
+        Read HDF5 result file and return pyPESTO result object.
+        """
+        sampleResult = {}
```

Comment:
```suggestion
        sample_result = {}
```
+ following occurrences